certified-connectors/Verified/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2681 @@
+            "x-ms-summary": "As Object",
+            "enum": [
+              true,
+              false
+            ]
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "url": {
+                  "type": "string",
+                  "title": "URL",
+                  "description": "File URL"
+                }
+              }
+            }
+          }
+        },
+        "x-ms-visibility": "important"
+      }
+    },
+    "/companies/{company-id}/users/{user-id}/settings": {
+      "get": {
+        "operationId": "GetCompaniesCompanyIdUsersUserIdSettings",
+        "summary": "Get user settings",
+        "description": "Gets a users settings by userid",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/CompanyId"
+          },
+          {
+            "name": "user-id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "User Id",
+            "description": "ID of the user in question",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/Setting"
+            }
+          }
+        },
+        "x-ms-visibility": "advanced"
+      }
+    },
+    "/envelopes/{envelope-id}": {
+      "get": {
+        "operationId": "GetEnvelopesEnvelopeId",
+        "summary": "Get envelope by id",
+        "description": "Gets envelope details by envelope id",
+        "tags": [
+          "cleaned"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/EnvelopeId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/Envelope"
+            }
+          }
+        },
+        "x-ms-visibility": "important"
+      },
+      "delete": {
+        "operationId": "DeleteEnvelopesEnvelopeId",
+        "summary": "Delete envelope by id",
+        "description": "Delets an envelope identified by its id",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Token"
+          },
+          {
+            "$ref": "#/parameters/ContentType"
+          },
+          {
+            "$ref": "#/parameters/XNamespace"
+          },
+          {
+            "$ref": "#/parameters/EnvelopeId"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
As stated in the API documentation this endpoint does not return any information in the body either
https://verified-dev.api-docs.io/v2.1/envelopes/envelope-delete